src/hub/dataload/sources/drugcentral/drugcentral_dump.py

```diff
@@ -1,15 +1,134 @@
+import csv
 import os
 
 import biothings
+import psycopg2
+
 import config
 
 biothings.config_for_app(config)
 
-from config import DATA_ARCHIVE_ROOT
-from biothings.hub.dataload.dumper import DummyDumper
+from biothings.hub.dataload.dumper import BaseDumper
 
+from config import DATA_ARCHIVE_ROOT, DRUGCENTRAL_PASSWORD
 
-class DrugCentralDumper(DummyDumper):
 
+class DrugCentralDumper(BaseDumper):
     SRC_NAME = "drugcentral"
     SRC_ROOT_FOLDER = os.path.join(DATA_ARCHIVE_ROOT, SRC_NAME)
+
+    # More info about the public DrugCentral Postgres database https://drugcentral.org/download
+    HOST = "unmtid-dbs.net"
+    PORT = 5433
+    DATABASE = "drugcentral"
+    USER = "drugman"
+
```

Comment:
@DylanWelzel a minor comment, if you want to make your Dumper class sort-of ready to be a generic PostgresqlDumper, you probably want to keep `PASSWORD` here, but set it to `DRUGCENTRAL_PASSWORD`, then reference it with generic `self.PASSWORD`.

Reply:
Updated in most recent commit f48f51b
```
from config import DATA_ARCHIVE_ROOT, DRUGCENTRAL_PASSWORD

        PASSWORD = DRUGCENTRAL_PASSWORD
    
        self.client = psycopg2.connect(
            database=self.DATABASE,
            user=self.USER,
            password=self.PASSWORD,
            host=self.HOST,
            port=self.PORT,
        )

```